### ISSUE TITLE
SecurePlayerSample Compilation Error

### ISSUE THREAD

**Issue by Mazyod** [ooyala/ios-sample-apps#86]
Hello,

Once I clone the repo and follow the steps to integrate the SecurePlayer SDK, I get a compilation error:

```
/Users/xxx/tmp/ios-sample-apps/AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Players/InsertAdPlayerViewController.m:51:48: No visible @interface for 'OOVASTAdSpot' declares the selector 'initWithTime:clickURL:trackingURLs:vastURL:'
```

Thanks,

**Comment by lental** [comment]
looking into it, thanks

**Comment by lental** [comment]
fixed in a3883206539777b1bc1692e425174c6e9e85fef5.  thanks for the catch
